packages/setup-remote-server/lib/commands/server.js

```diff
@@ -0,0 +1,475 @@
+		const intro = `${ format.title( `👋  Welcome to ${ packageName }` ) }
+
+This tool will guide you through the setup process of a new ${ format.comment( 'remote server' ) }.
+`;
+		log( intro );
+
+		const { isReady } = await inquirer.prompt( [
+			{
+				type: 'expand',
+				name: 'isReady',
+				default: 2, // default to help in order to avoid clicking straight through
+				choices: [
+					{ key: 'y', name: 'Yes', value: true },
+					{ key: 'n', name: 'No', value: false },
+				],
+				message: 'Has the server and project repo been setup for Deployer?',
+			},
+		] );
+
+		if ( ! isReady ) {
+			log( format.error( '\nAborted.' ) );
+			process.exit();
+		}
+
+		log();
+	}
+
+	// Check if project directory.
+	const dotLocalServer = `${ WORKING_DIR }/.local-server`;
+	let dotLocalServerExists;
+	try {
+		await fs.stat( dotLocalServer );
+		dotLocalServerExists = true;
+	} catch ( e ) {
+		dotLocalServerExists = false;
+	}
+
+	if ( dotLocalServerExists ) {
+		log(
+			format.error(
+				'This direcotry does not seem to be a project. Please run the command within a project directory.'
+			)
+		);
+		process.exit();
+	}
+
+	const deployYML = `${ WORKING_DIR }/deploy.yml`;
+	let deployYMLExists;
+	try {
+		fs.existsSync( deployYML );
+		deployYMLExists = true;
+	} catch ( err ) {
+		deployYMLExists = false;
+	}
+
+	if ( ! deployYMLExists ) {
+		log( format.error( 'This project does not seem to be setup for Deployer.' ) );
+		process.exit();
+	}
+
+	let deployYMLData;
+	try {
+		const deployYMLContents = await fs.readFileSync( deployYML, 'utf8' );
+		deployYMLData = YAML.parse( deployYMLContents );
+	} catch ( error ) {
+		log( error );
+	}
+
+	if ( ! deployYMLData ) {
+		log( format.error( 'Parsing deploy.yml failed.' ) );
+		process.exit();
+	}
+
+	const { remoteEnvoirnment } = await inquirer.prompt( [
+		{
+			type: 'list',
+			name: 'remoteEnvoirnment',
+			message: 'Choose the remote envoinrment',
+			choices: [ 'Staging', 'Production' ],
+			filter( value ) {
+				switch ( value ) {
+					case 'Production':
+						value = 'prod';
+						break;
+					case 'Staging':
+					default:
+						value = 'stage';
+						break;
+				}
+				return value;
+			},
+		},
+	] );
+
+	if ( deployYMLData[ remoteEnvoirnment ] === undefined ) {
+		log( format.error( 'The remote environment does not exist in deploy.yml.' ) );
+		process.exit();
+	}
+
+	const envoirnment = deployYMLData[ remoteEnvoirnment ].stage;
+
+	const { hostName, hostUser, privateKey } = await inquirer.prompt( [
+		{
+			type: 'input',
+			name: 'hostName',
+			default: deployYMLData[ '.base' ].hostname,
+			message: 'Enter the hostname for the remote server:',
+			validate: validateHostname,
+		},
+		{
+			type: 'input',
+			name: 'hostUser',
+			default: deployYMLData[ '.base' ].user,
+			message: 'Enter the host username:',
+			validate: validateSlug,
+		},
+		{
+			type: 'input',
+			name: 'privateKey',
+			default: '~/.ssh/id_rsa',
+			message: 'Enter the path for SSH Key:',
```

Comment:
This is the part for my local SSH key correct?

```suggestion
			message: 'Enter the local path to your private SSH key:',
```

It would be nice if this value is saved in the local config and used as the default on the next run.